### ISSUE TITLE
Sunshine 0.22.2 crashing Moonlight UWP on Xbox Series consoles

### ISSUE THREAD

**Issue by Trasen** [LizardByte/Sunshine#2351]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your issue described in the documentation?

- [X] I have read the documentation

### Is your issue present in the nightly release?

- [X] This issue is present in the nightly release

### Describe the Bug

When on a network with a 0.22.2 server, miliseconds after showing up in the UI the Moonlight app shuts down on Xbox Series X - no error message, as far as I can see there's no log - but it's consistent - only thing that changed was that I installed the latest version of Sunshine on one of the computers and the app stopped working.

I will ofc be posting an issue on the UWP app for Series too, guessing this could be under "intended behaviour" :) 

### Expected Behavior

Expecting the app not to crash and being able to connect to the server/ computer

### Additional Context

I've got 3 computers running Sunshine, everything worked great until I installed the Sunshine 0.22.2 server on my main computer.

### Host Operating System

Windows

### Operating System Version

22H2 Build 19045.4239

### Architecture

64 bit

### Sunshine commit or version

0.22.2

### Package

Windows - installer

### GPU Type

Nvidia

### GPU Model

3080Ti

### GPU Driver/Mesa Version

551.86

### Config

```Shell
sunshine_name = Stationära
origin_web_ui_allowed = pc
qp = 36
```

### Relevant log output

```shell
None
```

**Comment by Trasen** [comment]
reference to the issue on the moonlight-xbox app: https://github.com/TheElixZammuto/moonlight-xbox/issues/119

**Comment by Trasen** [comment]
Ok this is narly, i downgraded one release at a time, each time walking up to my living room, starting the Moonlight app on the Xbox in repeat until it didn't crash anymore. v0.19.1, confirmed that the Xbox app doesn't crash anymore, then i upgraded to 0.22.2 again, and now for whatever reason it works with the latest version.

**Comment by ReenigneArcher** [comment]
Thanks for the update. I'll close this as it probably won't be easy to replicate.

If you can figure out how to replicate the issue please let us know, and I'll re-open this.

**Comment by Trasen** [comment]
Yes, fresh install of latest Sunshine on a fresh install of Windows - that should do it

**Comment by ReenigneArcher** [comment]
You said it works with the latest version, so I'm skeptical.

I can't replicate with Xbox One X and nightly Sunshine.

**Comment by Trasen** [comment]
Yes after downgrading the server to 19 and then up again. Just up a VM, install Windows 10 and the latest version of Sunshine :) That should give the same scenario.
